raise RuntimeError(
    "The level_sets module has been moved to yt.data_objects.level_sets."
    "Please, change the import in your scripts from "
    "'from yt.analysis_modules.level_sets' to "
    "'from yt.data_objects.level_sets.'."
)
